gvsbuild/main.py

```diff
@@ -44,6 +45,9 @@
 from gvsbuild.build import build
 from gvsbuild.outdated import outdated
 
+# disable rich markup and highlight for log messages, do not assume any markup syntax
+rich.reconfigure(markup=False, highlight=False)
```

Comment:
If someone does have a / in an environment variable, and run with debug, then setting markup=False, but leaving highlight enabled will fix the issue without disabling the color functionality.